lib/plugins/aws/invoke/index.js

```diff
@@ -28,6 +28,17 @@ class AwsInvoke {
     // validate function exists in service
     this.options.functionObj = this.serverless.service.getFunction(this.options.function);
 
+    // if data is provided via CLI, sanitize it
+    if (this.options.data && typeof this.options.data === 'string') {
```

Comment:
Thanks! I'll fix this.

Not sure if it can be smth. different. /cc @eahefnawy @ac360